### PR TITLE
Add docker-sync support

### PR DESCRIPTION
**Pull request by kasperg**
- Add docker-sync.yml + docker-compose-dev.yml for docker-sync support
- Use PHP-FPPM + separate Apache container
- Move docroot to /var/www/web instead of /var/www/html
- Add default settings.php

This is basically a backport of most of the docker setup from the 
ding2-bbs branch with all of the project specific details removed.

### CODE REVIEW

**Comment by kasperg** [comment]
We should probably update the README as well before merging this.

**Review by rasben — APPROVED** [review]
👍

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,5 @@
 web/
 docker-compose.override.yml
 drupal.make
+.idea
```
Comment (kasperg) [inline .gitignore]: Please do not add `.idea` to project-specific `.gitignore`. If you use IntelliJ it belongs in [your global `.gitignore`](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore).

File: README.md [hunk 2]
```diff
@@ -27,11 +27,25 @@ Build the site with drush make:
...
+
+otherwise, you can use vanilla docker-compose:
+
 ```sh
 % docker-compose up
 ```
 
+or if you are using Mac, you can use docker-sync
+```
+% docker-sync-stack start
```
Comment (kasperg) [inline README.md]: This seems to be mentioned twice. Is that intentional?

File: docker/docker.settings.php [hunk 3]
```diff
@@ -67,3 +67,29 @@
 //   $conf['memcache_bins'] = array(
 //     'cache' => 'default',
 //   );
+
+
+// Configure memcache.
```
Comment (kasperg) [inline docker/docker.settings.php:72]: Based on this can we then remove the "Memcache setup" from the TODO list in the README?

**Review by kasperg — COMMENTED** [review]
I have a we comments to the changes I/you/we made @rasben

File: README.md (hunk 2 above)
Comment (rasben) [reply README.md]: Nope it wasnt - I've removed it now

File: .gitignore (hunk 1 above)
Comment (rasben) [reply .gitignore]: 👍

File: docker/docker.settings.php (hunk 3 above)
Comment (rasben) [reply docker/docker.settings.php:72]: Good point - i've removed it form the readme

**Pull request merged**